apps/omg_watcher/lib/omg_watcher/supervisor.ex

```diff
@@ -32,9 +32,15 @@ defmodule OMG.Watcher.Supervisor do
   end
 
   def init(:ok) do
+    db_root_path =
+      :omg_db
+      |> Application.fetch_env!(:path)
+      |> OMG.DB.root_path()
+
     {:ok, contract_deployment_height} = RootChain.get_root_deployment_height()
 
     children = [
+      OMG.DB.child_spec(db_path: Path.join([db_root_path, "exit_processor"]), name: OMG.DB.RocksDB.ExitProcessor),
```

Comment:
you're pulling in a child from another application into your supervision tree. can we find an alternative approach?

Reply:
The best way it'd be pass additional argument to the `:omg_db` application in Watcher's deps in `mix.exs`. But not sure it is possible. What other alternative can you see?